### DISCUSSION TITLE
Customize date format

### DISCUSSION THREAD

**Discussion by GilDev** [hay-kot/homebox#593] [Ideas]
The current date format seems to be YYYY-MM-DD when displayed, and MM/DD/YYYY when editing. This is not standard in multiple countries (also seems to make little sense because it’s not in order), and is different for displaying and editing.

It would be great to be able to customize this format and have it everywhere (both for displaying and editing).

**Comment by rubiojr** [comment]
:+1:! It'd be nice an option to use ISO 8601 dates, it's a bit harder for europeans otherwise :smile: 

![image](https://github.com/hay-kot/homebox/assets/10998/2428628e-2f1d-429d-b3fa-db554a49c80a)
